apps/api-gateway/src/user/dto/req.dto.ts

```diff
@@ -30,3 +30,23 @@ export class CreateUserDto {
   @IsNumber()
   public career_category_id: number;
 }
+
+export class UpdateUserDto {
+  @ApiProperty({ type: 'array', items: { type: 'number' } })
+  @IsArray()
+  @ArrayMaxSize(5)
+  @IsNumber({}, { each: true })
+  public stack_category_id?: number[];
+
+  @ApiProperty({ type: 'number' })
+  @IsNumber()
+  public position_category_id?: number;
+
+  @ApiProperty({ type: 'number' })
+  @IsNumber()
+  public project_category_id?: number;
+
```

Comment:
```suggestion
 @ApiProperty({ type: 'array', items: { type: 'number' } })
  @IsArray()
  @ArrayMaxSize(3)
  @IsNumber({}, { each: true })
  public project_category_id?: number [];

```
기획 의견을 반영해서 위의 타입으로 변경되어야 할 것 같습니다!